xtensor-ioConfig.cmake.in

```diff
@@ -16,14 +16,6 @@
 
 @PACKAGE_INIT@
 
-include(CMakeFindDependencyMacro)
-find_dependency(xtensor @xtensor_REQUIRED_VERSION@)
```

Comment:
Why removing this? It is required for linking with this transitive dependency.

Reply:
I was looking at xsimd's config. I will amend to only remove the checking on line 25.